examples/arc.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env python3
+#
+# Test for a push-button monome arc
+
+import asyncio
+import monome
+
+
+class ExampleArcApp(monome.ArcApp):
+    def __init__(self):
+        super().__init__()
+        self.pos = [0, 0, 0, 0]
+
+    def on_arc_ready(self):
+        print('Ready, clearing all rings...')
+        for n in range(0, 4):
+            self.arc.ring_all(n, 0)
+
+    def on_arc_disconnect(self):
+        print('Arc disconnected.')
+
+    def on_arc_delta(self, ring, delta):
```

Comment:
i tested this with an arc and see that there's no visible feedback when turning knobs counterclockwise.

if you're storing the current position in self.pos, what do you think about having the rings updated with a position indicator when turning the knobs in both directions?

Reply:
I literally got my arc and grid today (second hand)! I just wanted to check all the encoders, buttons and LEDs worked and as I am on windows using pymonome seemed the easiest way! 
As I had to write my own file for the arc I thought I would contribute back- it took me around 30 mins to get the arcs going whereas for the grid I had working examples already provided.

By all means we can do a more interesting/high quality example, but perhaps in a future PR? For now I have used a dim value for counter-clockwise, so even if you start with an counter-clockwise motion you still get immediate feedback.

Reply:
sure, and i appreciate the contribution. but i'm asking for a more intuitive behaviour for the following reasons:

- if another person runs the script, then turns the knob in CCW direction (without your latest changes) and doesn't see any LED feedback, they might think that there's a problem with the arc, serialosc or pymonome, rather than there's a problem with the script. it might take some time to figure this out.
- example scripts also have the purpose of demonstrating good pymonome programming patterns and style. your script looks good with the latest changes. i'm only having an additional suggestion to replace the loop for `ring_set` with `ring_range` (see below) and it's good to go!